sygnal/webpushpushkin.py

```diff
@@ -363,8 +365,6 @@ class HttpResponseWrapper:
     in the background.
```

Comment:
it now doesn't really happen "in the background", afaict? it's more that it is deferred until later.

How about:

>    To keep pywebpush happy we present it with some hardcoded values that
>    make its assertions pass even though the HTTP request has not yet been
>    made.

Reply:
Indeed, I forgot to update this as the async request was fired immediately before but now is deferred to the execute method :+1: Have adopted your suggestion.